Modules/gcmodule.c

```diff
@@ -326,7 +331,7 @@ gc_get_objects_impl(PyObject *module, Py_ssize_t generation)
     }
 
     PyInterpreterState *interp = _PyInterpreterState_GET();
-    return _PyGC_GetObjects(interp, (int)generation);
+    return _PyGC_GetObjects(interp, generation);
```

Comment:
GitHub's reporting lots of compiler warnings on this line on Windows: 
![image](https://github.com/user-attachments/assets/1c9bfcb7-d283-4036-afdd-8b4144247e62)